sql/hive/src/test/scala/org/apache/spark/sql/hive/MultiDatabaseSuite.scala

```diff
@@ -80,7 +80,7 @@ class MultiDatabaseSuite extends QueryTest with SQLTestUtils with TestHiveSingle
               |CREATE TABLE t1
               |USING parquet
               |OPTIONS (
-              |  path '$path'
+              |  path '${dir.toURI.toString}'
```

Comment:
Oh yes, right. I put the comment about path in the wrong place. I think it is now unused in other examples below though like line 241 but maybe not worth special casing.